homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -30,16 +35,82 @@
     DEPARTURE_TIME,
     DOMAIN,
     TIME_TYPES,
+    TRAFFIC_MODEL,
     TRANSIT_PREFS,
     TRANSPORT_TYPE,
     TRAVEL_MODE,
-    TRAVEL_MODEL,
     UNITS,
     UNITS_IMPERIAL,
     UNITS_METRIC,
 )
 from .helpers import InvalidApiKeyException, UnknownException, validate_config_entry
 
+NONE_SENTINEL = "none"
+
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_MODE): SelectSelector(
+            SelectSelectorConfig(
+                options=sorted(TRAVEL_MODE),
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key=CONF_MODE,
+            )
+        ),
+        vol.Optional(CONF_LANGUAGE, default=NONE_SENTINEL): SelectSelector(
+            SelectSelectorConfig(
+                options=[NONE_SENTINEL] + sorted(ALL_LANGUAGES),
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key=CONF_LANGUAGE,
+            )
+        ),
+        vol.Optional(CONF_AVOID, default=NONE_SENTINEL): SelectSelector(
+            SelectSelectorConfig(
+                options=[NONE_SENTINEL] + sorted(AVOID),
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key=CONF_AVOID,
+            )
+        ),
+        vol.Required(CONF_UNITS): SelectSelector(
+            SelectSelectorConfig(
+                options=sorted(UNITS),
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key=CONF_UNITS,
+            )
+        ),
+        vol.Required(CONF_TIME_TYPE): SelectSelector(
+            SelectSelectorConfig(
+                options=sorted(TIME_TYPES),
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key=CONF_TIME_TYPE,
+            )
+        ),
+        vol.Optional(CONF_TIME, default=""): cv.string,
+        vol.Optional(CONF_TRAFFIC_MODEL, default=NONE_SENTINEL): SelectSelector(
+            SelectSelectorConfig(
+                options=[NONE_SENTINEL] + sorted(TRAFFIC_MODEL),
```

Comment:
```suggestion
                options=[NONE_SENTINEL] + sorted(TRAFFIC_MODELS),
```

I suggest renaming the constant name of the options to plural. Some are already in plural, others not.